kbi/0008/index.rst

```diff
@@ -0,0 +1,106 @@
+.. index::
+   triple: git-annex; uninit; remove dataset annex
+
+KBI0008: Remove a dataset's annex
+=================================
+
+:authors: Christian Mönch <christian.moench@web.de>
+:discussion: https://github.com/datalad-handbook/book/issues/939
+:keywords: annex, remove annex, annex uninit
+
+This knowledge base item describes how to remove the annex from a DataLad
+dataset and propagate that change to siblings
+
+DataLad version used: this knowledge base item was created based on DataLad version 0.18.3
+
+
+The scenario
+------------
+
+You have a DataLad dataset that has annexed files and you decide that you
+don't want to use git-annex anymore in this dataset. This change should
+also be propagated to all siblings of the dataset.
+
+
+A few words of caution
+......................
+
+1. You should not remove the annex if you have large files, i.e. 1GB or larger, in your dataset. ``git`` is not intended for large files, which is the reason for the creation of ``git annex`` in the first place.
```

Comment:
```suggestion
1. You should not remove an annex of a dataset, if you have large files (>100MB), or many files (>several MB) that are cumulatively large. ``git`` is not intended for large files, which is the reason for the creation of ``git annex`` in the first place.
```

I am using the github file size limit as another arbitrary threshold. The original phrasing sound like 900MB is still OK -- but would actually already yield a fairly crippled repo.